src/eva/core/utils/operations.py

```diff
@@ -0,0 +1,26 @@
+"""Functional operations."""
+
+import re
+from typing import Iterable, List
+
+
+def numeric_sort(item: Iterable[str], /) -> List[str]:
```

Comment:
Can you write some simple unit tests for this?

If the strings contain multiple numbers at different places, it will use the first one for sorting right?

Reply:
yes add couple of unit tests!

it will be like this: `["a1b2", "a1b10", "a1b3"]` -> `["a1b2", "a1b3", "a1b10"]`

Reply:
Would be good to add a test case like `["a1b2", "a1b10", "a1b3", "a2b1", "a2b4"]`

Reply:
added this exactly one